tests/unit/test-waiter-test.ts

```diff
@@ -1,22 +1,46 @@
+// @ts-ignore
```

Comment:
Why do we need this ts-ignore? Doesn't this make it ignore the whole file? 

Is this perhaps because the `qunit` module is unknown to the system? If so, we should add a `types/qunit.d.ts` file instead of ts-ignoring...

Reply:
Yes, this was an issue with `@types/qunit`, in that it hadn't declared QUnit as a module. That's been fixed upstream in DefinitelyTyped/DefinitelyTyped/pull/35350. 

I can revisit adding the module, but was hoping to just pull in the fix from upstream.

Reply:
Ya, totally if they fixed it that seems great!